docs/api/Shell/dispose.md

```diff
@@ -9,5 +9,5 @@ Releases all resources used by the shell object and closes the PowerShell child_
 #### Example
 
 ```javascript
-ps.dispose().then(code => {}).catch(err => {});
+ps.dispose().then(code => {}).catch(error => {});
```

Comment:
I would generally recommend a disposer: https://stackoverflow.com/questions/28915677/what-is-the-promise-disposer-pattern

Reply:
lets talk about this case.